package mpsqual;

message ARINC717FrameRequest { 
	enum RequestTypeT {
		STOP = 0;
		RUN = 1;
		REPORT = 2;
	}
	
	required RequestTypeT requestType = 1 [default = STOP];
}

message ARINC717FrameResponse {
	enum AppStateT {
		STOPPED	= 0;
		RUNNING	= 1;
	}
	
	enum FrameSync {
		NO_SYNC = 0;
		SYNCED = 1;
	}
	
	required AppStateT state = 1;
	required FrameSync syncState = 2 [default = NO_SYNC];
	repeated int32 arinc717frame = 3;
}
